omisego-sdk/consumer-proguard-rules.pro

```diff
@@ -0,0 +1,25 @@
+# OMG
+-keep class co.omisego.omisego.** { *; }
+-keep interface co.omisego.omisego.** { *; }
+-keep class * implements co.omisego.omisego.** { *; }
```

Comment:
Do we really need to keep the whole SDK? 🤔 
I think the package containing the model classes should be enough no?
I don't see any particular reason not to minify OMGClient, security package etc... 🤔

Reply:
Fixed that 👍